src/test/java/edu/kis/powp/jobs2d/TestJobs2dApp.java

```diff
@@ -96,6 +97,11 @@ private static void setupLogger(Application application) {
         application.addComponentMenuElement(Logger.class, "OFF logging", (ActionEvent e) -> logger.setLevel(Level.OFF));
     }
 
+    private static void addSubscribers(){
+        UsageMonitorObserver usageMonitorObserver = new UsageMonitorObserver();
+        DriverFeature.getDriverManager().getChangePublisher().addSubscriber(usageMonitorObserver);
+    }
```

Comment:
the name of this method should concern the function you added. Those to lines could be added directly to setupDriver.method.